test/test.mli

```diff
@@ -1,11 +1,13 @@
 (*Ideally we should have a unified interface of testing framework, I am still thinking about how to abstract away the difference between JS testing and OUnit.*)
```

Comment:
Sure but we'd get a lot for free with bs-jest. It's pretty much complete as far as the needs we have here go. And it will probably (eventually) be more familiar to other contributors, unless we go properly in for bs-ava.

Then again, this isn't terribly complicated stuff anyway, so it doesn't matter all that much.